docs/user/custom_domains.rst

```diff
@@ -1,197 +0,0 @@
-  it is possible to validate the domain and provision the certificate before pointing your domain to Read the Docs.
-
-Certificate authority authorization
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Certificate authority authorization (CAA) is a security feature that allows domain owners to limit
-which certificate authorities (CAs) can issue certificates for a domain.
-This is done by setting CAA DNS records for your domain.
-CAA records are typically on the root domain, not subdomains
-since you can't have a CNAME and CAA record for the same subdomain.
-Here's an example for palletsprojects.com:
-
-.. prompt:: bash $, auto
-
-    $ dig CAA +short palletsprojects.com
-    0 issue "digicert.com"
-    0 issue "comodoca.com"
-    0 issue "letsencrypt.org"
-
-If there are CAA records for your domain that do not allow the certificate authorities that Read the Docs uses,
-you may see an error message like ``pending_validation: caa_error: YOURDOMAIN.TLD``
-in the Read the Docs dashboard for your domain.
-You will need to update your CAA records to allow us to issue the certificate.
-
-* For |org_brand|, we use Cloudflare which uses Digicert as a CA. See the `Cloudflare CAA FAQ`_ for details.
-* For |com_brand|, we use AWS Certificate Manager as a CA. See the `Amazon CAA guide`_ for details.
-
-.. _Cloudflare CAA FAQ: https://support.cloudflare.com/hc/en-us/articles/115000310832-Certification-Authority-Authorization-CAA-FAQ
-.. _Amazon CAA guide: https://docs.aws.amazon.com/acm/latest/userguide/setup-caa.html
-
-.. note::
-
-   If your custom domain was previously used in GitBook, contact GitBook support (via live chat in their website)
-   to remove the domain name from their DNS Zone in order for your domain name to work with Read the Docs,
-   else it will always redirect to GitBook.
-
-Canonical URLs
---------------
-
-Canonical URLs allow people to have consistent page URLs for domains.
-This is mainly useful for search engines,
-so that they can send people to the correct page.
-
-Read the Docs uses these in two ways:
-
-* We point all versions of your docs at default version, usually "latest" or "stable", as canonical.
-* We point at the user specified canonical URL, generally a custom domain for your docs.
-
-Example
-+++++++
-
-Fabric hosts their docs on Read the Docs.
-They mostly use their own domain for them ``http://docs.fabfile.org``.
-This means that Google will index both ``http://fabric-docs.readthedocs.io`` and
-``http://docs.fabfile.org`` for their documentation.
-
-Fabric will want to set ``http://docs.fabfile.org`` as their canonical URL.
-This means that when Google indexes ``http://fabric-docs.readthedocs.io``,
-it will know that it should really point at ``http://docs.fabfile.org``.
-
-Enabling
-++++++++
-
-You can set the canonical URL for your project in the Project Admin page.
-Check your :guilabel:`Admin` > :guilabel:`Domains` page for the domains that we know about.
```

Comment:
It seems we lost this subsection in the refactor.

Reply:
It's as a step when creating the domain https://github.com/readthedocs/readthedocs.org/pull/9266/files#diff-938211973d90543e901c5445def567dddb0542430d329c86285f88acf281e169R20